src/PresentationalComponents/RuleDetails/RuleDetails.js

```diff
@@ -41,146 +37,107 @@ class RuleDetails extends Component {
-                                            <Shield
-                                                hasTooltip={ false }
-                                                impact={ this.ruleResolutionRisk(rule) }
-                                                size={ 'md' }
-                                                title={ AppConstants.RISK_OF_CHANGE_LABEL[this.ruleResolutionRisk(rule)] || 'Undefined' }
-                                            />
-                                        </div>
-                                        <div className='pf-u-display-inline-flex'>
-                                            <span className={ `label pf-u-pl-sm ins-sev-clr-${this.ruleResolutionRisk(rule)}` }>
-                                                { AppConstants.RISK_OF_CHANGE_LABEL[this.ruleResolutionRisk(rule)] || 'Undefined' }
-                                            </span>
-                                        </div>
-                                    </div>
-                                    <p>{ AppConstants.RISK_OF_CHANGE_DESC[this.ruleResolutionRisk(rule)] }</p>
-                                </GridItem>
-                                { this.ruleRebootRequired && (<GridItem>
-                                    <span className='ins-sev-clr-4'>Restart Required</span>
-                                </GridItem>) }
-                            </Grid>
+                        <GridItem sm={ 4 } md={ 12 } className='pf-l-flex'>
+                            <div>Risk of Change</div>
+                            <div className='pf-u-display-inline-flex pf-m-align-center pf-u-pb-sm pf-u-pt-sm'>
+                                <div className='pf-u-display-inline-flex'>
+                                    <Shield
+                                        hasTooltip={ false }
+                                        impact={ this.ruleResolutionRisk(rule) }
+                                        size={ 'md' }
+                                        title={ AppConstants.RISK_OF_CHANGE_LABEL[this.ruleResolutionRisk(rule)] || 'Undefined' }
+                                    />
+                                </div>
+                                <div className='pf-u-display-inline-flex'>
+                                    <span className={ `label pf-u-pl-sm ins-sev-clr-${this.ruleResolutionRisk(rule)}` }>
+                                        { AppConstants.RISK_OF_CHANGE_LABEL[this.ruleResolutionRisk(rule)] || 'Undefined' }
+                                    </span>
+                                </div>
+                            </div>
+                            <p>{ AppConstants.RISK_OF_CHANGE_DESC[this.ruleResolutionRisk(rule)] }</p>
                         </GridItem>
+                        { rule.reboot_required && (<GridItem>
+                            <span className='ins-sev-clr-4'>Restart Required</span>
```

Comment:
To keep this consistent with other apps, can we use [Reboot](https://github.com/RedHatInsights/insights-frontend-components/blob/master/doc/components/reboot.md)?

Reply:
We have Restart on one page and Reboot on this one in our UXD mockups.  I vote for consistency.  :smile:

Reply:
https://redhat.invisionapp.com/share/V4NY3C6XKRW#/screens/344290759 

is the mocks we have saying Restart Required, can reword to reboot

Reply:
@AllenBW it also says Reboot on the page before that. When I asked Cam a few weeks ago, we're rolling with "Reboot"

Reply:
but Cam is 📤(out) 👅